### PR TITLE
PA-8333 added score error reason to the trip entity

### PR DESCRIPTION
**Pull request by SvitlanaLitovchenko**
#### :tickets: Jira Tickets
https://1mojio.atlassian.net/browse/PA-8333
#### :interrobang: What does this PR do?
added score error reason to the trip entity

### CODE REVIEW

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    ScoreErrorReason(String key) {
+        this.key = key;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public static ScoreErrorReason fromKey(String key) {
```
Comment (PavloZoria) [inline mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java:55]: I would recommend marking it with a `@Nullable` annotation in order to deliberately show nullability in kotlin code

Comment (SvitlanaLitovchenko) [reply mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java]: @PavloZoria, sounds good. Fixed. Thank you 👍

Comment (SvitlanaLitovchenko) [reply mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java:56]: Had to roll back the change as it caused a crash in the Transport class. 😞

**Review by oleksiiborys — APPROVED** [review]

**Review by PavloZoria — APPROVED** [review]

File: mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java (hunk 1 above)
Comment (SvitlanaLitovchenko) [reply mojio-sdk-model/src/main/java/io/moj/java/sdk/model/enums/ScoreErrorReason.java]: @PavloZoria, you were right, there was a wrong annotation. added org.jetbrains.annotations library. everything works well

**Review by oleksiiborys — APPROVED** [review]

**Pull request merged**